xml/System.Net.Sockets/SendPacketsElement.xml

```diff
@@ -255,11 +264,21 @@
         <Parameter Name="count" Type="System.Int32" Index="2" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="fileStream">To be added.</param>
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="fileStream">The file to be transmitted using the <see cref="M:System.Net.Sockets.Socket.SendPacketsAsync(System.Net.Sockets.SocketAsyncEventArgs)" /> method.</param>
+        <param name="offset">The offset, in bytes, from the beginning of the file to the location in the file to start sending the bytes in the file.</param>
+        <param name="count">The number of bytes to send starting from the <paramref name="offset" />. If <paramref name="count" /> is zero, the entire file is sent.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Sockets.SendPacketsElement" /> class using the specified <paramref name="fileStream" />, buffer <paramref name="offset" /> and <paramref name="count" />.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Net.Sockets.SendPacketsElement" /> class using the specified <range of a <see cref="T:System.IO.FileStream" /> object.</summary>
```

Reply:
If you want me to replace 
> the specified filestream, buffer offset and count.

for 

> the specified range of a filestream object.

Shouldn't we also apply that to ALL the rest of existing overloads, including the ones taking a `byte[]` and a `string` filename path? i.e: look at summary in the above overload:
https://github.com/dotnet/dotnet-api-docs/blob/54c925d4c4b0708415db50abfd0ea0bd36cc151a/xml/System.Net.Sockets/SendPacketsElement.xml#L225

Reply:
Yes, we should make the replacement elsewhere, @Jozkee.